### ISSUE TITLE
String instead of list in assembly_qc

### ISSUE THREAD

**Issue by nickjcroucher** [johnlees/PopPUNK#64]
Command was:

`./PopPUNK/poppunk-runner.py  --easy-run --r-files test.csv  --sketch-size 1000  --output test_out`

Error was:

```
PopPUNK (POPulation Partitioning Using Nucleotide Kmers)
	(with backend: sketchlib v1.1.0)
Mode: Creating clusters from assemblies (create_db & fit_model)
Traceback (most recent call last):
  File "./PopPUNK/poppunk-runner.py", line 10, in <module>
    main()
  File "/home/ncroucher/viruses/influenza/data/PopPUNK/PopPUNK/__main__.py", line 264, in main
    args.threads, args.overwrite)
  File "/home/ncroucher/viruses/influenza/data/PopPUNK/PopPUNK/sketchlib.py", line 315, in constructDatabase
    genome_length, max_prob = assembly_qc(sequences, klist, ignoreLengthOutliers)
  File "/home/ncroucher/viruses/influenza/data/PopPUNK/PopPUNK/utils.py", line 450, in assembly_qc
    "\n".join(outliers))
TypeError: sequence item 0: expected str instance, list found

```
This error was contingent upon the inclusion of a particular pair of sequences.

**Comment by nickjcroucher** [comment]
In util.py:

 ```
       outliers = []
        sigma = 5
        if not ignoreLengthOutliers:
            genome_length = np.mean(np.array(input_lengths))
            outlier_low = genome_length - sigma*np.std(input_lengths)
            outlier_high = genome_length + sigma*np.std(input_lengths)
            for length, name in zip(input_lengths, input_names):
                if length < outlier_low or length > outlier_high:
                    outliers.append(name)
            if outliers:
                sys.stderr.write("ERROR: Genomes with outlying lengths detected\n" +
                                 "\n".join(outliers))
                sys.exit(1)
```
"outliers" is a list of lists, so cannot be read by join.

**Comment by nickjcroucher** [comment]
Fixed in 2d941b7ebf11345acfd847d92b2c0244978dcdb0:

```
                sys.stderr.write("ERROR: Genomes with outlying lengths detected\n")
                for outlier in outliers:
                    sys.stderr.write('\n'.join(outlier) + '\n')
```

Not very Pythonic, but will do for now.
